java/src/main/java/ai/rapids/cudf/Scalar.java

```diff
@@ -539,6 +617,36 @@ public ColumnView getListAsColumnView() {
     return new ColumnView(getListAsColumnView(getScalarHandle()));
   }
 
+  /**
+   * Fetches views of children columns from struct scalar.
```

Comment:
The docs should state the caller is responsible for closing the resulting `ColumnView` instances or a native memory leak will occur.

Reply:
Yes! I added this notification into the doc.